.rubocop.yml

```diff
@@ -83,15 +85,15 @@ Layout/ParameterAlignment:
 Layout/SpaceAroundEqualsInParameterDefault:
   EnforcedStyle: 'no_space'
 
-Layout/SpaceAroundMethodCallOperator:
-  Enabled: true
-
 Layout/SpaceAroundOperators:
   Enabled: false
 
 Layout/SpaceBeforeBlockBraces:
   Enabled: false
 
+Layout/SpaceBeforeBrackets: # (new in 1.7)
```

Comment:
This cop seems to have issues, especially with situations like the example you found in our code – https://github.com/rubocop/rubocop/issues/9438, https://github.com/rubocop/rubocop/issues/9290

Maybe we should disable it for now?

Reply:
...wow that's impressive. I can do that, yes.